ci-operator/step-registry/openshift-qe/microshift-deploy/openshift-qe-microshift-deploy-commands.sh

```
@@ -0,0 +1,165 @@
+NODES=$(curl -sSk $OCPINV | jq -r ".nodes[0:${NUM_NODES}][].name")
+if [[ -z "${NODES}" ]]; then
+  echo "ERROR: No nodes returned from QUADS for lab cloud ${LAB_CLOUD}"
+  exit 1
+fi
+echo "Nodes to deploy MicroShift on: $NODES"
+
+# Copy SSH keys from bastion to provisioned nodes
+echo "Copying SSH keys to provisioned nodes..."
+for node in $NODES; do
+  echo "Copying SSH key to ${node}..."
+  ssh ${SSH_ARGS} root@${bastion} "
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)